examples/gateways/artifact-gateway-configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: artifact-gateway-configmap
+  namespace: argo-events
```

Comment:
do we need to specify the namespace here? in an example i would think ppl can deply to whatever namespace they choose to use.